frontend/packages/client/src/components/Community/hooks/useLinkValidator.js

```diff
@@ -16,30 +16,29 @@ export const urlPatternValidation = (url) => {
   return url === '' || !!regex.test(url);
 };
 const twitterValidator = (url) => {
-  const regex = new RegExp(
-    '^(?:https://)?(?:www\\.)?twitter\\.com/(\\w+)',
-    'i'
-  );
+  const regex = new RegExp('(https://)(www\\.)twitter\\.com/(\\w+)', 'i');
   return url === '' || !!regex.test(url);
 };
 const instagramValidator = (url) => {
   return (
     url === '' ||
-    /(?:(?:http|https):\/\/)?(?:www.)?(?:instagram.com|instagr.am|instagr.com)\/(\w+)/gim.test(
+    /(https:\/\/)(www\.)(?:instagram.com|instagr.am|instagr.com)\/(\w+)/gim.test(
       url
     )
   );
 };
 const discordValidator = (url) => {
   return (
     url === '' ||
-    /(https?:\/\/)?(www\.)?(discord\.(gg|io|me|li|com)|discordapp\.com\/invite)\/.+[a-zA-Z0-9]/gim.test(
+    /(https:\/\/)(www\.)(discord\.(gg|io|me|li|com)|discordapp\.com\/invite)\/.+[a-zA-Z0-9]/gim.test(
       url
     )
   );
 };
 const githubValidator = (url) => {
-  return url === '' || /https?:\/\/github\.com\/(?:[^/\s]+)/gim.test(url);
+  return (
+    url === '' || /(https:\/\/)(www\.)(github\.com)\/(?:[^/\s]+)/gim.test(url)
+  );
```

Comment:
changes done to enforce patter `https://www.` to be consistent with place holder

Reply:
The `www.` should still be optional in the regex